README.md

```diff
@@ -1705,10 +1711,10 @@ regarding stock awards and stock options.
   [Section 83(b)](https://www.law.cornell.edu/uscode/text/26/83), offers an alternative, called
   an
   [**83(b) election**](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/),
-  that will protect you from high potential tax at time of vesting, which ensures that
-  you’ll be taxed on the receipt of the “property” (the stock), rather than at the time the
-  stock vests. With an 83(b) election, you’re telling the IRS you want to pay taxes early,
-  and this can potentially reduce your tax significantly:
+  that protects employees from high potential tax at time of vesting, ensuring that they
+  will be taxed upon receipt of the property (the stock), rather than at the time the stock
+  vests. When an employee files an 83(b) election, they’re telling the IRS that they want to
```

Comment:
an employee -> a person (or someone?)

Or even "By filing an 83(b) election, you're telling the IRS that you want..."

I feel like with explanatory text (as opposed to the defining text) being so strict isn't necessary 100% of the time.

Reply:
It's still part of the definition here, technically. But, reverted to you...